### PR TITLE
PAYARA-2425 Fix for using multiple envrionment variable in a line

### PR DESCRIPTION
**Pull request by Cousjava**

### CODE REVIEW

**Comment by Cousjava** [comment]
Jenkins test please

**Comment by smillidge** [comment]
I assume this requires a 4.x PR as well

**Comment by mikecroft** [comment]
Jenkins test please

**Comment by Cousjava** [comment]
Jenkins test please

**Comment by mikecroft** [comment]
Note that the above error isn't resolved by the new commit. The problem is that you **can't add a JVM option which doesn't start with a hyphen**, not that it is parsed incorrectly. This is validation happening in the `create-jvm-options` command which needs to be tweaked slightly. (perhaps a check that, if the option does not start with a hyphen, then it checks to see if it is an environment variable, so only environment variables are permitted to not have hyphens).

**Comment by Cousjava** [comment]
Jenkins test please

File: nucleus/admin/launcher/src/main/java/com/sun/enterprise/admin/launcher/JvmOptions.java [hunk 1]
```diff
@@ -61,6 +61,12 @@
     private final static Pattern envP = Pattern.compile("([^\\$]*)\\$\\{ENV=([^\\}]*)\\}([^\\$]*)");
```
Comment (lprimak) [inline nucleus/admin/launcher/src/main/java/com/sun/enterprise/admin/launcher/JvmOptions.java]: the envP pattern needs a comment explaining what it does, with examples

**Comment by Cousjava** [comment]
Jenkins test please

**Pull request merged**